app/userclass.js

```diff
@@ -0,0 +1,12 @@
+<<<<<<< HEAD
+
+
+=======
```

Comment:
Fix the conflicts in file(with git mergetool and help from a trainer) and make sure the file doesn't contain conflict markers anymore